packages/selinux-policy/serge-testsuite/runtest.sh

```diff
@@ -104,9 +112,31 @@ rlJournalStart
         rlRun "sed -i 's/^expand-check[ ]*=.*$/expand-check = 0/' /etc/selinux/semanage.conf"
         if [ ! -d selinux-testsuite ] ; then
             rlRun "git clone $GIT_URL" 0
+            rlRun "pushd selinux-testsuite"
+            rlRun "git checkout $GIT_BRANCH" 0
+            for _ in $GIT_PULLS; do
+                rlRun "git config --global user.email nobody@redhat.com"
+                rlRun "git config --global user.name 'Nemo Nobody'"
+                rlRun "git checkout -b testing-cherry-picks" 0
+                break
+            done
+            for pull in $GIT_PULLS; do
+                ref="refs/pull/$pull/head"
+                if ! rlRun "git fetch origin $ref:$ref" 0; then
+                    rlRun "git checkout $GIT_BRANCH" 0
+                    rlWarn "PR merge failed, falling back to GIT_BRANCH"
```

Comment:
Are we sure we want to report warn here and below since it will mark the job as failing and send a failure report to developers? I'm thinking it might be better to just fall back (and log the message without warning), what do you think?